packages/api/package.json

```diff
@@ -41,7 +41,7 @@
     "it-last": "^2.0.0",
     "linkdex": "^3.0.0",
     "merge-options": "^3.0.4",
-    "multiformats": "^9.6.4",
+    "multiformats": "^13.1.1",
```

Comment:
needed to upgrade to use Link, which made needed to upgrade IPLD deps to decrease TS issues